chapter2/tests/t11/pool.d/slowww.conf

```diff
@@ -0,0 +1,466 @@
+; Start a new pool named 'www'.
+; the variable $pool can be used in any directive and will be replaced by the
+; pool name ('www' here)
+[slowww]
+
+; Also added for the correct execution of the slow report
+php_admin_value[max_execution_time] = 605
```

Comment:
prefer to keep every `php_*` directive at the bottom of the page.

Reply:
remove all other options you don't need (or defaults). So it's easier to see where slow.conf is different vs www.conf

Reply:
> prefer to keep every `php_*` directive at the bottom of the page.

@dmitry-mightydevops 
Done